.github/workflows/pr.yaml

```diff
@@ -24,6 +24,22 @@ jobs:
     - name: Test
       run: make test
 
+    - name: Run Gosec Security Scanner
+      run: |
+        go install github.com/securego/gosec/v2/cmd/gosec@v2.18.2
+        make gosec
+        if [[ $? != 0 ]]
```

Comment:
Do we need this check? If the exit code of `make gosec` is not 0, the action should fail, no?

Reply:
yes, it's redundant since I added the check in the make rule.  I'll remove it